StripeFinancialConnections/StripeFinancialConnections/Source/API Bindings/FinancialConnectionsAPIClient.swift

```diff
@@ -555,7 +555,7 @@ extension FinancialConnectionsAPIClient: FinancialConnectionsAPI {
                 return self.post(
                     resource: APIEndpointAttachPaymentAccount,
                     parameters: body,
-                    useConsumerPublishableKeyIfNeeded: false
+                    useConsumerPublishableKeyIfNeeded: true
```

Comment:
@tillh-stripe could you sanity check that we're OK to use the consumer's publishable key for these requests? They all can happen after a Link user is verified.

Reply:
That looks good to me!